index.js

```diff
@@ -41,7 +41,14 @@ module.exports = (rootDirs, delay = 50) => {
 		}
 	});
 
-	return emitter;
```

Comment:
Technically, this is a breaking change, because before that we exposed the entire emitter interface including `once`, `removeListener` etc. – but I guess that's not that important?

Reply:
Good point. I didn't want the hassle of using `Proxy`, but also didn't want to monkey-patch the `emitter` instance - but you're right, it's potentially misleading.

Another option would be to be explicit(ly constrained) and only expose an object with nothing but `#onEdit`, `#onError` and `#terminate` rather than a pseudo-emitter?

I suppose going `Proxy` would be the Right Way™ though?

Reply:
I think, we should do it like you do it right now. If we notice that people need more parts of the interface of the emitter, then we can still add that later. I would however document the interface of the object we pass back to the user.